doc/website: new design

elad-n opened the pull request.

a8m commented:
Thanks for the help with this @elad-n! Added 3 comments, but besides this, the design looks fantastic. 

Header links are broken.
![image](https://user-images.githubusercontent.com/7413593/149371588-d8abe079-d7e4-47f9-82e5-86a1c076919e.png)

Footer links are too big and the indentation is weird on small screens. 
![image](https://user-images.githubusercontent.com/7413593/149371758-0f869089-edb9-4246-9325-96509b87997b.png)
![image](https://user-images.githubusercontent.com/7413593/149372025-6a98f8ed-9e2d-4966-a166-4747f83c278f.png)

a8m commented:
Also, the logo is 44K in size. Please, move it out from this repo to CDN (cc @rotemtam).

a8m commented on doc/website/docusaurus.config.js:
```diff
@@ -6,7 +6,7 @@ module.exports = {
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
+  favicon: 'img/logoAtlas.svg',
```
45KB size image for the favicon?

elad-n replied:
replaced to a ico file.

a8m commented:
Why both logoAtlas.svg and logo.svg are needed? It's the same file

elad-n commented:
> Why both logoAtlas.svg and logo.svg are needed? It's the same file

ya, the first one should have been an ico format. i replaced it. so now its ok.
now we have one for the favicon and one for the navbar  image

elad-n commented:
> Thanks for the help with this @elad-n! Added 3 comments, but besides this, the design looks fantastic.
> 
> Header links are broken. ![image](https://user-images.githubusercontent.com/7413593/149371588-d8abe079-d7e4-47f9-82e5-86a1c076919e.png)
> 
> Footer links are too big and the indentation is weird on small screens. ![image](https://user-images.githubusercontent.com/7413593/149371758-0f869089-edb9-4246-9325-96509b87997b.png) ![image](https://user-images.githubusercontent.com/7413593/149372025-6a98f8ed-9e2d-4966-a166-4747f83c278f.png)



> Thanks for the help with this @elad-n! Added 3 comments, but besides this, the design looks fantastic.
> 
> Header links are broken. ![image](https://user-images.githubusercontent.com/7413593/149371588-d8abe079-d7e4-47f9-82e5-86a1c076919e.png)
> 
> Footer links are too big and the indentation is weird on small screens. ![image](https://user-images.githubusercontent.com/7413593/149371758-0f869089-edb9-4246-9325-96509b87997b.png) ![image](https://user-images.githubusercontent.com/7413593/149372025-6a98f8ed-9e2d-4966-a166-4747f83c278f.png)

The header link won't
![Screen Shot 2022-01-13 at 22 43 57](https://user-images.githubusercontent.com/9443953/149406584-e06578d3-905b-4d61-b5c4-eb6d164013d1.png)
![Screen Shot 2022-01-13 at 22 44 05](https://user-images.githubusercontent.com/9443953/149406592-f5f130d4-d8dc-4b51-a098-0198e47c1ce5.png)
 work on localhost no ? 
the rest is fixed.

rotemtam commented:
Thanks for working on this @elad-n !

Some feedback. 

Before the "hero" image loads, site looks like:
<img width="1175" alt="Screen Shot 2022-01-15 at 12 13 09" src="https://user-images.githubusercontent.com/1522681/149618132-136eb2b9-122d-4c3c-823e-70417ff47ea8.png">
Let's pick a bg color more similar to the image so it doesn't look weird.
---
<img width="1064" alt="Screen Shot 2022-01-15 at 12 14 37" src="https://user-images.githubusercontent.com/1522681/149618171-a8a10d35-0228-4597-8628-c4800e149588.png">

H1 is centered.. this looks odd, can we revert it to the way it normally looks? 

---

<img width="490" alt="Screen Shot 2022-01-15 at 12 16 02" src="https://user-images.githubusercontent.com/1522681/149618212-68a945e0-5db1-4233-ad11-e82ef9e007d3.png">
Navbar, should look like the design.  Spacing from top to text/icons is off, and text should be aligned to the right (see how it is in the design). 

Footer, discord link takes me to 404.

rotemtam commented on doc/website/src/pages/index.js:
```diff
@@ -0,0 +1,134 @@
+                   </h2>
+                   <p className={index.paragraph2}
+                      style={{ color: "#757889" }}>
+                       With a simple command line flag, Atlas opens up as a rich user interface to visualize your data topology and manage your data topology.
```
```suggestion
                       With a single command-line flag, Atlas opens up as a rich user interface to visualize and manage your data topology.
```

a8m commented on doc/website/docusaurus.config.js:
```diff
@@ -19,68 +19,44 @@ module.exports = {
+          "items": [
+            {"label": "Getting Started", "to": "cli/getting-started/setting-up"},
+            {"label": "Data Definition Language ", "to": "ddl/intro"},
+            {"label": "CLI Reference", "to": "CLI/atlas"},
```
```suggestion
            {"label": "CLI Reference", "to": "cli/atlas"},
```

why the URL is uppercased?
